hearts/hearts.py

```diff
@@ -13,11 +13,79 @@ def __hash__(self):
 
 
 class Game(object):
-    def __init___(self, players, points_to_win=100):
-        self.points_to_win = points_to_win
+    def __init__(self, players, points_to_win=100):
+        self.max_points = points_to_win
         self.players = players
-        self.round_counter = 1
+        self.rounds = []
+        self.round_number = 1
+
         shuffle(self.players)
+        self.create_round()
+
+    def create_round(self):
+        pass_direction = {0: 'keep', 1: 'left', 2: 'right', 3: 'across'}
+        new_round = Round(self.players, pass_direction[self.round_number % 4])
+        self.rounds.append(new_round)
+
+    @property
+    def scores(self):
+        # Returns a dict of the form {round_number: {Player: score}}
+        scores = {1: {player: 0 for player in self.players}}
```

Comment:
`scores` should be a list whose entries are the dicts. Any time you have a dictionary whose keys are sequential integers, this is a sign you instead want a list.